cpu/Makefile.include.gnu

```diff
@@ -9,6 +9,11 @@ endif
 export AS = $(PREFIX)as
 export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
-export OBJCOPY = $(PREFIX)objcopy
+ifneq ("$(wildcard $(PREFIX)objcopy)","")
```

Comment:
`OFLAGS` is copied from `boards/native/Makefile.include`. I'm not sure this is needed.